### ISSUE TITLE
[A11y]: contrast of selected item within a toolbar is not enough

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/whiteboard#872]
<img width="1273" height="169" alt="Image" src="https://github.com/user-attachments/assets/5354adc8-2bf9-448a-8439-549556b57870" />

 For some buttons within a toolbar there is not visible focus at all, for others focus state was implemented right:

<img width="1273" height="169" alt="Image" src="https://github.com/user-attachments/assets/0344bb87-cd8a-468e-9cd5-7cccf2512ab8" />
